modal/image.py

```diff
@@ -1038,8 +1036,8 @@ def from_dockerhub(
         force_build: bool = False,
         **kwargs,
     ):
-        f"""{_from_dockerhub_deprecation_msg}"""
-        deprecation_error(date(2023, 8, 25), _from_dockerhub_deprecation_msg)
+        """`Image.from_dockerhub` is deprecated. Use `Image.from_registry` instead."""
+        deprecation_error(date(2023, 8, 25), "`Image.from_dockerhub` is deprecated. Use `Image.from_registry` instead.")
```

Comment:
```suggestion
        deprecation_error(date(2023, 8, 25), Image.from_dockerhub.__doc__)
```

Is the way we usually do this.